pretix_wallet/static/pretix_wallet/css/wallet.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+    font-family: "Montserrat";
+    src: url('/static/pretix_wallet/webfonts/Montserrat-VariableFont_wght.ttf') format('truetype'),
```

Comment:
this may break at some point because the static URL is hardcoded instead of using the one from the django settings

Reply:
Well, this is ultimately a question of which design crime you want to commit: CSS in the HTML (so it can be templated) or hardcoded URL in the CSS file, isn't it?

Reply:
i guess you could fix that with SCSS that needs to be compiled anyways, but I think its fine for now